views/js/runner/plugins/tools/textToSpeech/textToSpeech.js

```diff
@@ -239,12 +239,28 @@ define([
              * Click to pronounce
              */
             clickToSpeak: function clickToSpeak() {
+                var self = this;
+
                 options.enableClickToSpeak = !options.enableClickToSpeak;
 
                 this._exec('enableClickToSpeak', options.enableClickToSpeak);
                 this._exec('enableContinuousReading', !options.enableClickToSpeak); // continuous reading is off when click to speak is on
                 this.trigger('clickToSpeak');
 
+                //adding each item a special class by presence of which normal click handling could be prevented and passed to click-to-speak handling
+                $('.qti-item').each(function() {
```

Comment:
The TestRunner provider exposes an areaBroker object that gives access to the DOM. Please use it.

```javascript
testRunner.getAreaBroker().getContentArea()
```

Reply:
Thanks! Didn't know that we can access it through brokers. Will fix this.